staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go

```diff
@@ -53,7 +55,9 @@ type AnnotateOptions struct {
 	// Common user flags
 	overwrite       bool
 	local           bool
-	dryrun          bool
+	dryRunStrategy  cmdutil.DryRunStrategy
+	dynamicClient   dynamic.Interface
```

Comment:
The fact that we have to add these two everywhere makes me a little sad :-/

Reply:
As discussed, let's replace these with `*resource.DryRunVerifier`s.

The tests failed because i forgot to plumb the dynamic + discovery clients.

Reply:
Fixed!